table/data_block_hash_index.h

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) 2011-present, Facebook, Inc. All rights reserved.
+//  This source code is licensed under both the GPLv2 (found in the
+//  COPYING file in the root directory) and Apache 2.0 License
+//  (found in the LICENSE.Apache file in the root directory).
+
+#pragma once
+
+#include <string>
+#include <vector>
+
+namespace rocksdb {
+
+// The format of the datablock hash map is as follows:
```

Comment:
1. Lets mention here clearly that this is an experimental feature that is being added to improve point-lookup cost.
2. Lets also mention that this Index is per block, and will live within the data-block, to avoid people confusing with the per-table Index blocks.